modules/programs/i3status-rust.nix

```diff
@@ -21,36 +21,32 @@ in {
           blocks = mkOption {
             type = settingsFormat.type;
             default = [
+              { block = "cpu"; }
               {
                 block = "disk_space";
                 path = "/";
-                alias = "/";
                 info_type = "available";
-                unit = "GB";
-                interval = 60;
+                interval = 20;
                 warning = 20.0;
                 alert = 10.0;
+                format = " $icon root: $available.eng(w:2) ";
               }
               {
                 block = "memory";
-                display_type = "memory";
-                format_mem = "{mem_used_percents}";
-                format_swap = "{swap_used_percents}";
+                format = " $icon $mem_total_used_percents.eng(w:2) ";
+                format_alt = " $icon_swap $swap_used_percents.eng(w:2) ";
               }
               {
-                block = "cpu";
-                interval = 1;
+                block = "sound";
+                click = [{
+                  button = "left";
+                  cmd = "pavucontrol";
+                }];
               }
-              {
-                block = "load";
-                interval = 1;
-                format = "{1m}";
-              }
-              { block = "sound"; }
               {
                 block = "time";
-                interval = 60;
-                format = "%a %d/%m %R";
+                interval = 5;
+                format = " $timestamp.datetime(f:'%a %d/%m %R') ";
```

Comment:
Are the spaces needed?

Reply:
Yes.

Reply:
You can check the release notes: https://github.com/greshake/i3status-rust/blob/master/NEWS.md#i3status-rust-0300

> Icons are now part of the format string option as a placeholder in blocks where format is customisable. If you have modified format and would like to keep the same behaviour (icon, whitespace) you need to update the value. For example,

```
[[block]]
block = "cpu"
format = "{utilization}"
```

needs to be changed to:

```
[[block]]
block = "cpu"
format = " $icon $utilization "
```